src/manifests_workflow/input_manifests.py

```diff
@@ -106,22 +106,23 @@ def update(self, min_klass: Union[Type[ComponentOpenSearchMin], Type[ComponentOp
                 self.add_to_cron(release_version)
 
     def create_manifest(self, version: str, components: List = []) -> InputManifest:
-        image_map = {
-            "opensearch": "opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2",
-            "opensearch-dashboards": "opensearchstaging/ci-runner:ci-runner-centos7-opensearch-dashboards-build-v2"
-        }
+        templates_base_path = os.path.join(self.manifests_path(), "templates")
+        template_version_folder = version.split(".")[0] + ".x"
+        template_full_path = os.path.join(templates_base_path, self.prefix, template_version_folder,
+                                          "manifest.yml")
+        if os.path.exists(template_full_path):
+            input_manifest_templates = InputManifest.from_file(open(template_full_path))
+        else:
+            input_manifest_templates = InputManifest.from_file(open(os.path.join(templates_base_path, self.prefix,
+                                                                                 "2.x", "manifest.yml")))
```

Comment:
This hard-coding is bound to cause problems. 

1. Use the entire manifest, not just the `templates.ci` part, so remove the default `data`. 
2. Add a default folder for the default manifest and load that instead of 2.x/manifest.yml.